### PR TITLE
Try to clarify some wording.

### PR DESCRIPTION
**Pull request by rodwiddowson**
I stumbed over the documentation for the Bitness attribute of the Component element.

I have tried to clean it up but I am unsure what was meant by the first sentence I edited.

Hope this helps

### CODE REVIEW

**Comment by rodwiddowson** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/xsd4/wix.xsd [hunk 1]
```diff
@@ -8145,10 +8145,10 @@
       <xs:attribute name="Bitness" type="BitnessTypeUnion">
         <xs:annotation>
           <xs:documentation>
-            Overrides the default Component bitness. Only 64-bit Components an install to 64-bit locations such
+            Overrides the default Component bitness. Only 64-bit Components install to 64-bit locations such
             as `ProgramFiles64Folder` and the 64-bit registry. The value `always64` will force the Component
             bitness to be 64-bit and cannot be included in 32-bit packages.
-            Simliarly, the value `always32` will force the Component bitness to 32-bit and can be included in
+            Simliarly, the value `always32` will force the Component bitness to 32-bit. 32-bit Components be included in
```
Comment (robmen) [inline src/xsd4/wix.xsd:8151]: The grammar was better before this edit.

File: src/xsd4/wix.xsd [hunk 2]
```diff
@@ -8145,10 +8145,10 @@
       <xs:attribute name="Bitness" type="BitnessTypeUnion">
         <xs:annotation>
           <xs:documentation>
-            Overrides the default Component bitness. Only 64-bit Components an install to 64-bit locations such
+            Overrides the default Component bitness. Only 64-bit Components install to 64-bit locations such
```
Comment (robmen) [inline src/xsd4/wix.xsd:8148]: I think that `an` was supposed to be `can`. I think that's the better fix.

File: src/xsd4/wix.xsd (hunk 1 above)
Comment (rodwiddowson) [reply src/xsd4/wix.xsd:8151]: > The grammar was better before this edit.

Actually I think its more the  grammar its meaning.  What your (correct) words mean is "You can include the attribute 'always32' in [wix files intended to be built into] in 64 bit packages",  I read it is a more anodyne "32 bit Components can be included in 64 installations but 64 bit components cannot".  I get (on rereading a much better sense of the constraints in your version

I missed this - this distinction is subtle but important,

It is getting late here.  I'll try to do a revised PR (with one additional character) tomorrow.  Thanks

**Pull request merged**